### PR TITLE
feat: network change

### PR DESCRIPTION
**Pull request by nesquikm**
## Description

Add connections storage, bloc and bottom sheet for changing connections.

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: lib/app/service/nekoton_related/connection_service/network_presets.dart [hunk 1]
```diff
@@ -1,50 +1,65 @@
...
     networkType: NetworkType.ever,
   ),
 ];
 
 /// Presets for venom network
-const venomPresets = [
+final _venomPresets = [
   // venom is in test, uncomment it later
   // ConnectionData.jrpc(
+  //   id: 'preset_venom_mainnet_jrpc',
```
Comment (Alex-A4) [inline lib/app/service/nekoton_related/connection_service/network_presets.dart:54]: This is not jrpc, this should be proto

File: integration_test/migration_test.dart [hunk 2]
```diff
@@ -296,8 +294,6 @@ void main() {
         _publicKey: [_address],
       },
     );
-    expect(await storage.readCurrentConnection(), _connection);
-    expect(storage.currentConnection, _connection);
```
Comment (Alex-A4) [inline integration_test/migration_test.dart:300]: Why did you just remove migrating current connection?

File: lib/feature/add_seed/add_seed_enable_biometry/view/add_seed_enable_biometry_view.dart [hunk 3]
```diff
@@ -55,7 +55,7 @@ class AddSeedEnableBiometryView extends StatelessWidget {
           text: LocaleKeys.useSomething.tr(
             args: [
               // ignore: prefer-trailing-comma
```
Comment (Alex-A4) [inline lib/feature/add_seed/add_seed_enable_biometry/view/add_seed_enable_biometry_view.dart:57]: I think, we can remove ignore line

File: lib/app/service/nekoton_related/connection_service/network_presets.dart (hunk 1 above)
Comment (nesquikm) [reply lib/app/service/nekoton_related/connection_service/network_presets.dart]: 🤟

File: integration_test/migration_test.dart (hunk 2 above)
Comment (nesquikm) [reply integration_test/migration_test.dart:300]: Reverted.

File: lib/feature/add_seed/add_seed_enable_biometry/view/add_seed_enable_biometry_view.dart (hunk 3 above)
Comment (nesquikm) [reply lib/feature/add_seed/add_seed_enable_biometry/view/add_seed_enable_biometry_view.dart]: 🤟

**Review by Alex-A4 — APPROVED** [review]

**Pull request merged**